### ISSUE TITLE
Gratitude

### ISSUE THREAD

**Issue by bitic** [kaorahi/howm#12]
This is a simple message to show gratitude to the howm author and the community around it, because the howm task management system is so clever that has made my life much easier. I have restarted using Emacs just for it! Thanks!

**Comment by rhstanton** [comment]
Yes, I just discovered the package and love it. It reminds me of an old MSDOS program I used to use, and really like, about 35 years ago(!) called Zoomracks.

**Comment by Vedrinic** [comment]
It's really wonderful. 
Wish there could be a mobile app.

**Comment by kaorahi** [comment]
FYI: #36

**Comment by Vedrinic** [comment]
> FYI: [#36](https://github.com/kaorahi/howm/issues/36)

Wow I didn't know it existed. I am gonna try it out.
Thank you so much for all.
